dkms.in

```diff
@@ -1460,7 +1460,7 @@ do_uninstall()
                 dir_to_remove="${dir_to_remove#/}"
             done
             # dir_to_remove may be an empty string
-            if [[ -d "$dir_to_remove" ]]; then
+            if [[ -d "$install_tree/$1/$dir_to_remove" ]]; then
```

Comment:
I don't think this is a proper way to fix #276. 

The error message `rmdir: failed to remove '': No such file or directory` mentioned in #244 occurs because `find_actual_dest_module_location` may output nothing (one possibility: #275), causing the variable `dir_to_remove` to be an empty string.

When the `dir_to_remove` variable is an empty string, your fix will cause that error message to appear again.

Reply:
Just removing the `-d` is enough IMO

Reply:
Implemented.